.github/workflows/ci.yml

```
@@ -45,23 +45,21 @@ jobs:
 
       - name: Upload coverage
         uses: codecov/codecov-action@v3
-        if: |
-          !startsWith(matrix.os, 'ubuntu')
+        if: matrix.os != ubuntu-22.04
```

Looks like a syntax error in actions' yaml. You can see the error at the bottom of this page https://github.com/denoland/deno_std/actions/runs/4349140842

```suggestion
        if: matrix.os != 'ubuntu-22.04'
```